iosApp/Modules/XCFrameworks/debug/shared.xcframework/Info.plist

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These files shouldn't be here.  The shared frameworks shouldn't be pointing to these files anymore anyway, so you should be able to delete the iosApp/XCFrameworks folder in Finder and the app should still build.